src/odemis/gui/model/__init__.py

```diff
@@ -880,6 +880,11 @@ def __init__(self, name, stage=None, stream_classes=None, fov_hw=None):
         else:
             self.stream_classes = stream_classes
         self._stage = stage
+        # TODO fix it
+        if isinstance(self.stream_classes, tuple):
```

Comment:
It checks if it is a tuple because self.stream_classes can be a tuple or just a single class, not a list or tuple of classes.